### ISSUE TITLE
setPolicy failed when i set the policyType to Thermal

### ISSUE THREAD

**Issue by yilongx** [intel/NMPRK#25]
setPolicy failed when i set the policyType to Thermal,  what's the real range of policy trigger value?
## test code as follow:

// create policy test !
 nmprk::translation::policy_t Setpolicy;
 Setpolicy.policyId = 0x03;
 Setpolicy.component = nmprk::translation::domainSystem;
 Setpolicy.policyType = nmprk::translation::policyThermal; 
 //Setpolicy.policyType = nmprk::translation::policyPower;
 Setpolicy.policyLimit = 350;
 Setpolicy.statReportingPeriod = 3600;
 Setpolicy.correctionTime = 60000;
 Setpolicy.policyEnabled = true;
 Setpolicy.sendAlert = false;
 Setpolicy.shutdown = false;
## the error message as follows:

Attempting to create policy ID = 3..... Failed!
Caught Exception setPolicy(0x03): Code[134] MSG[Policy trigger value out of range]

**Comment by jrmassoni** [comment]
Unfortunately there is no such thing as a policyThermal or policyPower, these were incorrectly created in the original code.  This byte actually controls the triggering mechanism.  These have actually been replaced by the following flags:
   noPolicyTrigger, 
   inletTempTrigger,
   missingPowerTimeout, 
   afterHostResetTrigger,
   bootTimePolicy  
This will be in the updated documentation.  The purpose for these flags is to set up the type of triggering.  The policyThermal or inletTempTrigger was tested in the update testTranslation code and I verifed that it did work.  That once the temperature that I set was passed, the policy was enabled.  If you find this not to be true, then let me know.  The question is should I remove the words policyPower and policyThermal in the code.  Currently it is kept for backward compatibilty.  They both are in fact power policies.  policyPower actually means there is no trigger policy and policyThermal actually means to use a temperature to trigger the power policy.
